app/presenters/email_alert_presenter.rb

```diff
@@ -24,6 +25,19 @@ def body
     end
   end
 
+  def tags
+    { format: document.publishing_api_document_type }.deep_merge(metadata)
+  end
+
+  def metadata
```

Comment:
The reason I went with a dynamic approach was because for each document type, the tags (with the exception of format) are different. So I copied this pattern from document presenter.